### PR TITLE
Begin condensing update_book_keeping_matrix function

### PR DESCRIPTION
**Pull request by OatmealSoldier**
Should we consider changing turnovers_per_game_taken and turnovers_per_game_given_up to turnovers_per_game_gained and turnovers_per_game_allowed to keep it consistent with the other features?

### CODE REVIEW

**Comment by yiskylee** [comment]
@SEStarkman @emersonboyd Not a football fan, do turnovers_allowed and turnovers_gained sound natural?

**Comment by SEStarkman** [comment]
I'm not sure.  Turnovers are mistakes, so I don't know if it is really allowed and gained so much as given up and taken.

**Comment by emersonboyd** [comment]
I agree with Sam on this one. "Given" and "taken" are more clear.

File: FeatureGenerator.py [hunk 1]
```diff
@@ -118,47 +71,142 @@ def update_book_keeping_matrix(book_keeping_matrix, book_keeping_matrix_column,
     #             num_games_played_home_team + 1)
     #     book_keeping_matrix.ix[home_team, 'num_games_played'] += 1
 
-    elif book_keeping_matrix_column == 'rush_yards_per_game_allowed_away_team' or book_keeping_matrix_column == 'rush_yards_per_game_allowed_home_team':
+    # elif book_keeping_matrix_column == 'rush_yards_per_game_allowed_away_team' or book_keeping_matrix_column == 'rush_yards_per_game_allowed_home_team':
```
Comment (yiskylee) [inline FeatureGenerator.py]: Could you take out the comments and maybe copy it to some scratch files in case they will be needed?

**Comment by yiskylee** [comment]
Hey @OatmealSoldier , have you read that git merge tutorial I sent recently and ready to take on merging master to your branch?

**Pull request merged**